README.md

```diff
@@ -4,6 +4,81 @@ This repository contains:
 
 - an [example proposal](./src/contracts/AddEmissionAdminPayload.sol) payload which could be used to setup liquidity mining on a governance controlled aave v3 pool
 - a [test](./tests/EmissionTestOpOptimism.t.sol) simulating the configuration of certain assets to receive liquidity mining
+- a [test](./tests/EmissionConfigurationTestMATICXPolygon.t.sol) simulating the setting up of new configuration of certain assets after the liquidity mining program has been created
+
+## Instructions to activate Liquidity Mining on Aave V3:
+
+<img width="924" alt="Screenshot 2023-04-10 at 11 27 10 AM" src="https://user-images.githubusercontent.com/22850280/230836420-7b5c4bba-d851-4258-90c6-602d33eaf845.png">
+
+1. Make sure the rewards funds that are needed to be distributed for Liquidity Mining are present in the Rewards Vault.
+
+2. Do an ERC-20 approve of the total rewards to be distributed to the Transfer Strategy contract, this is contract by Aave which helps to pull the Liquidity Mining rewards from the Rewards Vault address to distribute to the user. To know more about how Transfer Strategy contract works you can check [here](https://github.com/aave/aave-v3-periphery/blob/master/docs/rewards/rewards-transfer-strategies.md).
+
+   _Note: The Emission Admin is an address which has access to manange and configure the reward emissions by calling the Emission Manager contract and the    general type of Transfer Strategy contract used for Liquidity Mining is of type PullRewardsStrategy._
+
+3. Finally we need to configure the Liquidity Mining emissions on the Emission Manager contract from the Emission Admin by calling the `configureAssets()` function which will take the array of the following struct to configure liquidity mining for mulitple assets for the same reward or multiple assets for mutiple rewards.
+
+   ```
+   EMISSION_MANAGER.configureAssets([{
+
+     emissionPerSecond: The emission per second following rewards unit decimals.
+
+     totalSupply: The total supply of the asset to incentivize. This should be kept as 0 as the Emissions Manager will fill this up.
+
+     distributionEnd: The end of the distribution of rewards (in seconds).
+
+     asset: The asset for which rewards should be given. Should be the address of the aave aToken (for deposit) or debtToken (for borrow).
+            In case where the asset for reward is for debt token please put the address of stable debt token for rewards in stable borrow mode
+            and address of variable debt token for rewards in variable borrow mode.
+
+     reward: The reward token address to be used for Liquidity Mining for the asset.
+
+     transferStrategy: The address of transfer strategy contract.
+
+     rewardOracle: The Chainlink Aggregator compatible Price Oracle of the reward.
```

Comment:
Better to be explicit that this is not really used on-chain, it is for off-chain infra like UIs to have price conversion